devel-common/src/tests_common/pytest_plugin.py

```diff
@@ -224,6 +224,12 @@ def pytest_addoption(parser: pytest.Parser):
         dest="db_init",
         help="Forces database initialization before tests",
     )
+    group.addoption(
+        "--without-db-init",
+        action="store_true",
+        dest="no_db_init",
+        help="Forces NO database initialization before tests",
```

Comment:
This feels redundant with `--with-db-init`. If `--with-db-init` is not set then no DB initialization should happen. Why do we need this flag?

Reply:
Yeah, there is a very confusing venn diagram here of possible paths. Really with should be called "force-db-init" since it only forces it to happen, it can happen when that thing is set to false (which you would think means force do not, but no, it's only force in the true case). And so mine is the opposite, covering that case. It forces no db-init in the pytest plugin (not even evaluating other paths that could lead to inits).

I know it's confusing, hopefully that makes sense!

Reply:
I see, that makes sense :) Could you please update the `help` of both commands to basically explain what you just explain, it could very helpful for others (or even us in some time xD). Could you also mention they are mutually exclusive (might be obvious but ...)